src/backend/sideload/games.ts

```diff
@@ -163,7 +177,9 @@ export async function launchApp(appName: string): Promise<boolean> {
         logWarning('File not executable, changing permissions temporarilly', {
           prefix: LogPrefix.Backend
         })
-        await chmod(executable, 0o775)
+        if (!isMac && !executable.includes('app')) {
```

Comment:
Shouldn't the `!isMac` be enough here? The `.includes` could lead to false positives, and I don't think you can run MacOS' .app files on other OSs anyways

Reply:
No, it is not enough because you can sideload other things on mac. Not only .app.
So it will be on mac and needs to have .app on the name, if there it not. A bin, a python script or anything else, we would not run that.

Reply:
I'm confused. This condition is saying "Run `chmod` if we're not on MacOS & the executable path does not contain `app`". That does not sound like what you describe there.

Seems to me like swapping the `&&` to an `||` *would* be what you want though (always run it if we're not on MacOS, or if we are on MacOS, run it if the target's not an `app`)

Reply:
Yes, you are right. fixed now.